types/src/config.rs

```diff
@@ -288,6 +298,45 @@ pub struct VestingSchedule {
     pub cliff_period: Duration,
     /// Vesting duration period (e.g., 18 months)
     pub vesting_period: Duration,
+    /// An optional instant claim percentage that can be claimed right after the sale ends.
+    /// `10000 = 100%`
+    pub instant_claim: Option<u16>,
```

Comment:
The name is ambiguous, I suppose. The value of the variable with such a name assumes a number of tokens rather than a percentage. I would propose to rename it to `init_claim_percentage`.

Reply:
For me `init_claim` even more unclear.

Reply:
Then `instant_claim_percentage`?